android-auto/src/main/java/com/mapbox/examples/androidauto/car/MainCarAppService.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.examples.androidauto.car
+
+import androidx.car.app.CarAppService
+import androidx.car.app.Session
+import androidx.car.app.validation.HostValidator
+
+class MainCarAppService : CarAppService() {
+    override fun createHostValidator(): HostValidator {
+        return HostValidator.ALLOW_ALL_HOSTS_VALIDATOR
+        // TODO limit hosts for production
+        //    https://github.com/mapbox/1tap-android/issues/988
```

Comment:
Remove this.

Reply:
Thinking about this one.. I think we want to move all the android-auto issues into the public repo then

Reply:
Sounds good to me 🚀

Reply:
Github transfer issues making it painful 🤔 

![Screen Shot 2021-08-11 at 11 17 16 AM](https://user-images.githubusercontent.com/3021882/129081963-1c8124ba-7fe3-413b-9efd-190c05e49950.png)

Reply:
Oh wait.. can't transfer from private to public lol.. damnit

Reply:
Yeah, there's a copy function though.